src/components/AddExisitingStudentsModal.tsx

```diff
@@ -81,34 +83,60 @@ const AddExistingStudentsModal: React.FC<AddExistingStudentsModalProps> = ({ onC
       .catch(err => console.log(err))
   }
 
-  const studentsNotInStudentGroup = studentDocuments
-    ?.filter(student => {
-      let studentIsInClass = false
-      studentsInThisStudentGroupDocuments?.forEach(studentInGroup => {
-        if (studentInGroup.studentId === student.docId) {
-          studentIsInClass = true
-        }
-      })
-      return !studentIsInClass
-    })
-    .map(doc => {
-      return (
-        <StudentPreview
-          key={doc.docId}
-          student={doc}
-          selectedStudentsToAdd={selectedStudentsToAdd}
-          setSelectedStudentsToAdd={setSelectedStudentsToAdd}
-        />
-      )
+  const studentsNotInStudentGroup = studentDocuments?.filter(student => {
+    let studentIsInStudentGroup = false
+    studentsInThisStudentGroupDocuments?.forEach(studentInGroup => {
+      if (studentInGroup.studentId === student.docId) {
+        studentIsInStudentGroup = true
+      }
     })
+    return !studentIsInStudentGroup
+  })
+
+  const selectAllHandler = (e: React.SyntheticEvent) => {
```

Comment:
We don't have to use synthetic events anymore per https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript

Reply:
That's a cool article, but I don't see a part that discusses no longer needing synthetic events.  What should it be typed as?